supplementary/alica_ros1/alica_tests/src/test/test_blackboard.cpp

```diff
@@ -7,8 +7,6 @@
 
 namespace alica
 {
-namespace
```

Comment:
hope this change won't break ODR

Reply:
its working fine

Reply:
ODR violation can work randomly technically

Reply:
yes, but ODR will mostly occur if anonymous namespaces are used in headers. Here it should not matter, because this is a cpp file with mostly tests & test fixtures. Only 3 classes are defined here & all of them are exclusive to this cpp file, so there should be no ODR violations

Reply:
I told about types definitions. It's much more unpleasant problem, here I prepared a simple example https://github.com/ntfshard/cpp-odr-violation-types-example

But here, I checked and seems it shouldn't be a problem.

Reply:
ok, thank you for the example. I'll keep this in mind

Could you please approve the PR if it is fine?